pkg/framework/plugins/podlifetime/pod_lifetime.go

```diff
@@ -85,6 +85,13 @@ func New(args runtime.Object, handle frameworktypes.Handle) (frameworktypes.Plug
 				return true
 			}
 
+			// Init Container Status Reason
+			for _, containerStatus := range pod.Status.InitContainerStatuses {
```

Comment:
[RemoveFailedPods](https://github.com/kubernetes-sigs/descheduler/blob/master/README.md#removefailedpods) and [RemovePodsHavingTooManyRestarts](https://github.com/kubernetes-sigs/descheduler/blob/master/README.md#removepodshavingtoomanyrestarts) both offer the flag `includingInitContainers` so to be consistent we should include that flag [ref](https://github.com/kubernetes-sigs/descheduler/blob/master/pkg/framework/plugins/removepodshavingtoomanyrestarts/toomanyrestarts.go#L85-L89). thoughts?

Reply:
updated, however does increase the size of the PR and I would argue that including init containers is likely expected behaviour for erroring statuses

Reply:
Understood. If we didn't guard it by a new flag, then it could break existing functionality for users.